ifcb-parsing-processing/requirements.txt

```diff
@@ -0,0 +1,9 @@
+pandas
+re
+os
+pytz
+datetime
+chardet
+geopy
+csv
+jupyter
```

Comment:
These packages should be removed because they are in the python standard library and will throw an error in the installation process as written: re, os, datetime, csv